src/worker/mod.rs

```diff
@@ -539,7 +584,7 @@ impl Worker {
                     query_responses,
                     sticky_attributes: None,
                     return_new_workflow_task: false,
-                    force_create_new_workflow_task: false,
+                    force_create_new_workflow_task: force_new_wft,
```

Comment:
return_new_workflow_tasks should be true here, right?
Is that part of the coming heartbeat PR?

Reply:
Yes it will be :)